packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -613,29 +629,34 @@ class NestedScrollViewState extends State<NestedScrollView> {
 
   @override
   Widget build(BuildContext context) {
+    final ScrollPhysics _scrollPhysics = widget.physics?.applyTo(const ClampingScrollPhysics())
+      ?? widget.scrollBehavior?.getScrollPhysics(context).applyTo(const ClampingScrollPhysics())
+      ?? const ClampingScrollPhysics();
+
     return _InheritedNestedScrollView(
       state: this,
-      child: Builder(
-        builder: (BuildContext context) {
-          _lastHasScrolledBody = _coordinator!.hasScrolledBody;
-          return _NestedScrollViewCustomScrollView(
-            dragStartBehavior: widget.dragStartBehavior,
-            scrollDirection: widget.scrollDirection,
-            reverse: widget.reverse,
-            physics: widget.physics != null
-              ? widget.physics!.applyTo(const ClampingScrollPhysics())
-              : const ClampingScrollPhysics(),
-            controller: _coordinator!._outerController,
-            slivers: widget._buildSlivers(
-              context,
-              _coordinator!._innerController,
-              _lastHasScrolledBody!,
-            ),
-            handle: _absorberHandle,
-            clipBehavior: widget.clipBehavior,
-            restorationId: widget.restorationId,
-          );
-        },
+      child: ScrollConfiguration(
+        behavior: widget.scrollBehavior ?? ScrollConfiguration.of(context).copyWith(scrollbars: false),
```

Comment:
This will disable scrollbars for all descendant scrollable. Is that what we want here? Or should we only disable it for the _NestedScrollViewCustomScrollView?